index.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>Here and in section title for simulation above, should it be "of&nbsp;<strong>the&nbsp;</strong>OPD"? Also in the text there are times I think it should be <strong>the </strong>OPD?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>